test/perlobj.jl

```diff
@@ -150,8 +150,10 @@
 
         p.MILP = @pm Polytope.MixedIntegerLinearProgram( LINEAR_OBJECTIVE = obj, INTEGER_VARIABLES = intvar)
 
-        # segfaults when called from different thread, see
-        # https://github.com/oscar-system/Polymake.jl/issues/144
-        @test p.MILP.MINIMAL_VALUE == -7
+        if ENV["OMP_NUM_THREADS"] == 1
```

Comment:
of course this will fail when `OMP_NUM_THREADS` is not defined, we need to `get(ENV, "OMP_NUM_THREADS", 1)` it

thanks @saschatimme for thorough review :D (no offence, I noted this only late in the evening :P)